<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HelpForm</class>
 <widget class="QWidget" name="HelpForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>755</width>
    <height>583</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTextBrowser" name="textBrowser">
   <property name="geometry">
    <rect>
     <x>-5</x>
     <y>0</y>
     <width>761</width>
     <height>551</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pbPrev">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>552</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <underline>true</underline>
    </font>
   </property>
   <property name="text">
    <string>Назад</string>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pbHome">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>550</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <underline>true</underline>
    </font>
   </property>
   <property name="text">
    <string>На главную</string>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pbNext">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>550</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <underline>true</underline>
    </font>
   </property>
   <property name="text">
    <string>Вперёд</string>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
